### ISSUE TITLE
What does "remark" mean in occurrence search?

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#651]
**What does &quot;remark&quot; mean in occurrence search?**

Is this the number of issues found? SHoudl we then not call these issues? Also this record here is listed as having 3 remarks, but if you go to the details one issue tag shows:
https://www.gbif.org/occurrence/1676599206

-----
fbitem-4cd51ab3c35a7b3305c9ec1422d26bc93a21a482

System: Chrome 62.0.3202 / Mac OS X 10.12.6
Referer: https://www.gbif.org/occurrence/search?taxon_key=1878862&advanced=1
Window size: width 1680 - height 888
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-08T09:47:52.662Z',mode:absolute,to:'2017-11-08T09:53:52.662Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-08T09:47:52.662Z',mode:absolute,to:'2017-11-08T09:53:52.662Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MortenHofft** [comment]
Yeah, it makes sense to align the naming. I'm not sure if I prefer remarks over issues. The thing is that we use the issue flags to indicate both problems and comments. E.g. NO_TAXON_MATCH is an issue, but COORDINATES_ROUNDED is a remark.

The previous portal always showed all issues in the header. That meant that almost all occurrences would have a list of issues in the header, but most of them would be comments. To me that meant that you ended up ignoring them because they didn't matter in most cases. 

Instead we introduced a severity of issues. On [the occurrence](https://www.gbif.org/occurrence/1676599206) we only show the actual issues with a severity of warning or error in the header. 

![screen shot 2017-11-08 at 11 21 53](https://user-images.githubusercontent.com/6759267/32543948-644c0b2e-c477-11e7-825d-a811757a5691.png)

Later we show all the issues/remarks. For example that we decided to round the coordinates to 5 decimals.

![screen shot 2017-11-08 at 11 21 49](https://user-images.githubusercontent.com/6759267/32543967-7062e18a-c477-11e7-9183-183aca4978a3.png)

Perhaps a solution could be to rename issues to remarks, and then use the word issues for remarks with a severity of warning or error?

A shame that the user didn't leave any contact info
